### DISCUSSION TITLE
bad alignment and design

### DISCUSSION THREAD

**Discussion by lara95777** [woocommerce/woocommerce#44894] [Checkout Flow]
I keep running in to problems with the alignment after updates with woo blocks cart+chekout. I had to previously add CSS to fix a corrupt update writing the text over my input field boxes. Now the mobile cart is causing problems no customization options either, before update there was a nice floating bar at bottom of cart to 'proceed to chekout' this has been removed why? Now another alignment issue this time in the mobile cart the shipping selection button is ontop of the text. It's really annoying having to go fix all these things with CSS when they should be editable (and were working fine in the first place).

**Comment by senadir** [comment]
Hey! I'm not seeing any alignment issues on my end, you can provide a screenshot?
